go.mod

```diff
@@ -1,19 +1,21 @@
-module github.com/giantswarm/statusresource/v2
+module github.com/giantswarm/statusresource/v3
 
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions/v2 v2.0.0
+	github.com/giantswarm/apiextensions/v3 v3.4.0
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/errors v0.2.3
 	github.com/giantswarm/exporterkit v0.2.0
-	github.com/giantswarm/k8sclient/v4 v4.0.0
+	github.com/giantswarm/k8sclient/v5 v5.0.0
 	github.com/giantswarm/microerror v0.2.1
-	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/micrologger v0.3.3
 	github.com/giantswarm/operatorkit/v2 v2.0.0
 	github.com/giantswarm/tenantcluster/v3 v3.0.0
```

Comment:
Needs to be updated to v4 I guess

Reply:
Thanks missed that one. Done.

Reply:
I didn't even notice `operatorkit`. I meant `tenantcluster` should be v4 :) Funny coincidence.

Reply:
Wow so I missed 2!  I think I've looked at too many go.mod files today. 😄 

Should be good this time. PTAL

Reply:
I know the feeling...